### PR TITLE
test: Added test for `Table.remove_rows_with_outliers` to test multiple outliers in one column and outliers in two different columns

### LINKED ISSUE #274: Pytest can't detect 2 or more outliers in a function
### Describe the bug

The following function: _**def remove_rows_with_outliers**_ doesn't perform its function correctly because **it can only detect and remove a maximum of 1 outlier**. That is, 2 or more outliers should be removed but that doesn't work.

### To Reproduce

1.  Run the following function: **_test_should_remove_rows_with_outliers_** in _/Stdlib/tests/safeds/data/tabular/containers/table/test_remove_rows_with_outliers.py_ 

> The Tests should run successfully and remove only 1 row containing an outlier

2.  Add 1 or more outliers to an existing column e.g _col1, col2 or col3._ 

> Outlier Definition:  An outlier is defined as a value that has a distance of more than 3 standard deviations from the column mean.  Missing values are not considered outliers. They are also ignored during the calculation of the standard deviation.

3. Repeat step 1

### Expected behavior

_All Outliers should be detected and removed._

### Screenshots (optional)

_No response_

### Additional Context (optional)

_No response_

### PR DESCRIPTION
**Pull request by Marsmaennchen221**
### Summary of Changes

test: Added test for `Table.remove_rows_with_outliers` to test multiple outliers in one column and outliers in two different columns

See my [Comment](https://github.com/Safe-DS/Stdlib/issues/274#issuecomment-1554492317) in the referenced issue for further explanation as there is no bug with the current method `Table.remove_rows_with_outliers`

### Additional Context

See #274

### CODE REVIEW

**Comment by Marsmaennchen221** [comment]
Waiting for either #308 or #123 for the improved `Table.__eq__` method for the comparison of the empty tables.

**Review by alex-senger — APPROVED** [review]
LGTM

File: tests/safeds/data/tabular/containers/_table/test_remove_rows_with_outliers.py [hunk 1]
```diff
@@ -78,7 +250,14 @@
...
     ],
-    ids=["no outliers", "with outliers", "no rows", "empty"],
+    ids=[
+        "no outliers",
+        "one outlier",
+        "outliers in two different columns",
+        "multiple outliers in one column",
+        "no rows",
+        "empty",
+    ],
```
Comment (jxnior01) [inline tests/safeds/data/tabular/containers/_table/test_remove_rows_with_outliers.py:260]: LGTM. All edge cases covered.

**Review by jxnior01 — APPROVED** [review]
